newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -27,88 +26,21 @@ export type DataSource = Array<
     |}
 >;
 
-const styles = {
+const styles = makeStyles({
   container: {
     position: 'relative',
   },
-  inputRoot: {
-    flexWrap: 'wrap',
-  },
-  inputInput: {
-    width: 'auto',
-    flexGrow: 1,
-  },
-  menuPopper: {
-    // Ensure the popper is above everything (modal, dialog, snackbar, tooltips, etc).
-    // There will be only one AutoComplete opened at a time, so it's fair to put the
-    // highest z index. If this is breaking, check the z-index of material-ui.
-    zIndex: muiZIndex.tooltip + 100,
-  },
-  menuPaper: {
-    marginTop: 8,
-    // Limit the size of the menu:
-    maxHeight: 250,
-    overflowY: 'auto',
-  },
-};
-
-function renderTextField(textFieldProps) {
-  const { InputProps, ...other } = textFieldProps;
-
-  return (
-    <TextField
-      InputProps={{
-        style: styles.inputRoot,
-        ...InputProps,
-      }}
-      // eslint-disable-next-line react/jsx-no-duplicate-props
-      inputProps={{
-        style: styles.inputInput,
-      }}
-      {...other}
-    />
-  );
-}
-
-function renderItem(itemProps) {
-  const { item, index, menuItemProps, highlightedIndex, selected } = itemProps;
-
-  if (item.type === 'separator') {
-    return <Divider key={'separator-' + index} />;
+  root: {
+    flexWrap: "wrap",
+    '& .MuiAutocomplete-inputRoot .MuiAutocomplete-input': {
+      flexGrow: 1,
+      width: "auto",
+    }
   }
+});
 
-  const isHighlighted = highlightedIndex === index;
 
-  return (
-    <MenuItem
-      {...menuItemProps}
-      dense
-      key={
-        item.value
-          ? 'item-with-value-' + item.value
-          : 'item-without-value' + index
-      }
-      selected={isHighlighted}
-      component="div"
-      style={{
-        fontWeight: selected ? 500 : 400,
-      }}
-    >
-      {item.renderIcon && <ListItemIcon>{item.renderIcon()}</ListItemIcon>}
```

Comment:
Is your updated version showing icons? If not, it's a must have to keep in the new version :) 
According to what I see in the docs, maybe something like `renderOption` or something else to customize the rendering of the items.